core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -283,6 +341,35 @@ oppia.controller('ExplorationEditor', [
       });
     };
 
+    $scope.showWelcomeExplorationModal = function() {
+      var modalInstance = $modal.open({
+        templateUrl: 'modals/welcomeExploration',
+        backdrop: true,
+        controller: [
+          '$scope', '$modalInstance', function($scope, $modalInstance) {
+            $scope.beginTutorial = $modalInstance.close;
+
+            $scope.cancel = function() {
+              $modalInstance.dismiss('cancel');
+            };
+          }
+        ],
+        windowClass: 'oppia-welcome-modal'
+      });
+
+      modalInstance.result.then(function() {
+        $scope.startTutorial();
+      }, function() {
+        stateEditorTutorialFirstTimeService.markTutorialFinished();
+      });
+    };
+
+    $scope.openEditorTutorialFirstTime = function() {
```

Comment:
markTutorialFinished is used in EditorServices.js. It triggers a popover the first time the tuorial is skipped or finished on the help button that says something like "Click here for help in the future." It also sets _currentlyInFirstVisit to false